AmplifyPlugins/Core/AWSPluginsCoreTests/Model/GraphQLRequest/GraphQLRequestSyncCustomPrimaryKeyTests.swift

```diff
@@ -14,6 +14,7 @@ class GraphQLRequestSyncCustomPrimaryKeyTests: XCTestCase {
 
     override func setUp() {
         ModelRegistry.register(modelType: CustomerOrder.self)
+        ModelRegistry.register(modelType: CustomerWithDateInPK.self)
```

Comment:
nit: would prefer this to be a in a different file like `GraphQLRequestSyncCustomPrimaryKeyWithDateTests`

Reply:
CustomerOrder was written at a very different time, whereas CustomerWithDateInPK was written with V2 transform. Even though this is a unit test, there's an aspect of this where we will most likely provision the backend to continue verifying integration tests. also if we want to add another CPK use case, most likely it will be in another test class so it scales